lib/TBDGen/TBDGen.cpp

```diff
@@ -56,16 +56,28 @@ using namespace llvm::yaml;
 using StringSet = llvm::StringSet<>;
 using SymbolKind = llvm::MachO::SymbolKind;
 
+static constexpr StringLiteral ObjC2ClassNamePrefix = "_OBJC_CLASS_$_";
+static constexpr StringLiteral ObjC2MetaClassNamePrefix = "_OBJC_METACLASS_$_";
+
 static bool isGlobalOrStaticVar(VarDecl *VD) {
   return VD->isStatic() || VD->getDeclContext()->isModuleScopeContext();
 }
 
+// If a symbol is implied, we don't need to emit it explictly into the tbd file.
+// e.g. When a symbol is in the `objc-classes` section in a tbd file, a additional
+// symbol with `_OBJC_CLASS_$_` is implied.
+static bool isSymbolImplied(StringRef name) {
```

Comment:
Instead, please just comment out these two calls elsewhere in this file:

```
addSymbol(LinkEntity::forObjCClass(CD));
addSymbol(LinkEntity::forObjCMetaclass(CD));
```

Reply:
We do need them in the symbol sets for verifying against irgen.